### ISSUE TITLE
loss function

### ISSUE THREAD

**Issue by Zhang-bd** [Baisonm-Li/HSR-KAN#1]
Hello, I want to commend you on the exceptional work you have done on hyperspectral image super-resolution.  I observed that the paper mentions the use of sparse loss, but I couldn't locate the corresponding code implementation in the 'main.py' file.  Could you kindly provide some insight into the reason behind this discrepancy?  
Thank you for your attention.

**Comment by Baisonm-Li** [comment]
`main.py `  is a general training script designed for convenient comparison with other methods. If you want to train KANFormer, please appropriately invoke `regularization_loss` within the `KANFormer` module. We will update the code structure soon to improve readability and reproducibility. Thank you for your interest in our work.

**Comment by Baisonm-Li** [comment]
> Hello, I want to commend you on the exceptional work you have done on hyperspectral image super-resolution. I observed that the paper mentions the use of sparse loss, but I couldn't locate the corresponding code implementation in the 'main.py' file. Could you kindly provide some insight into the reason behind this discrepancy? Thank you for your attention.

`main.py` is a general training script designed for convenient comparison with other methods. If you want to train KANFormer, please appropriately invoke `regularization_loss` within the `KANFormer` module. We will update the code structure soon to improve readability and reproducibility. Thank you for your interest in our work.
